bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/events/ThingTriggerEvent.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.core.thing.events;
+
+import org.eclipse.smarthome.core.events.AbstractEvent;
+import org.eclipse.smarthome.core.types.Type;
+
+/**
+ * {@link ThingTriggerEvent}s can be used to deliver triggers through the Eclipse SmartHome event bus.
+ * Trigger events must be created with the {@link ThingEventFactory}.
+ */
+public class ThingTriggerEvent extends AbstractEvent {
+
+    /**
+     * The item state event type.
+     */
+    public final static String TYPE = ThingTriggerEvent.class.getSimpleName();
+
+    /**
+     * The event.
+     */
+    private final Type event;
+
+    /**
+     * Constructs a new item state event.
+     *
+     * @param topic the topic
+     * @param payload the payload
+     * @param source the source, can be null
```

Comment:
imho it would be good to clarify here that the topic is related to the thing, while the source identifies the channel which received trigger